packaging/macosx/darktable-nightly.bundle

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0"?>
```

Comment:
i had to create a copy of the original bundle file as some dependencies seem to be packaged slightly differently when using homebrew. see diff between original bundle and this one for more details.